### DISCUSSION TITLE
Can't manage sponsorship in mobile

### DISCUSSION THREAD

**Discussion by MQ-J** [community/community#34027] [Mobile]
There is no place to see my sponsorship information

**Comment by Iron-Ham** [comment]
Hi @MQ-J! 

Thanks for reporting.
We don't currently support sponsorships on mobile, but I can make an internal tracking issue for the team to discuss. Out of curiosity, what would you like to be able to do with sponsorships on mobile?

**Reply by MQ-J** [reply]
I'd like to manage my tiers in mobile, and see the others too. I'm beginner in sponsorship, and checking another sponsors profile give me some ideas.

Also, just thinking, imagine if we can use Google Play Payments to support anyone's project, using the mobile app?
